tests/RepositoryGeneratorTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace RonasIT\Support\Tests;
+
+use RonasIT\Support\Events\SuccessCreateMessage;
+use RonasIT\Support\Events\WarningEvent;
+use RonasIT\Support\Exceptions\ClassNotExistsException;
+use RonasIT\Support\Generators\RepositoryGenerator;
+use RonasIT\Support\Tests\Support\Repository\RepositoryMockTrait;
+
+class RepositoryGeneratorTest extends TestCase
+{
+    use RepositoryMockTrait;
+
+    public function testModelNotExist()
+    {
+        $this->mockClass(RepositoryGenerator::class, [
+            $this->classExistsMethodCall(['models', 'Post'], false),
+        ]);
+
+        $this->assertExceptionThrew(
+            className: ClassNotExistsException::class,
+            message: "Cannot create Post Model cause Post Model does not exists. "
```

Comment:
please check exception message, seems we need to throw next text

```suggestion
            message: "Cannot create PostRepository cause Post Model does not exists. "
```